proxy/logging/LogField.cc

```diff
@@ -772,7 +772,9 @@ LogFieldList::marshal_len(LogAccess *lad)
   int bytes = 0;
   for (LogField *f = first(); f; f = next(f)) {
     if (f->type() != LogField::sINT) {
-      bytes += f->marshal_len(lad);
+      const int len = f->marshal_len(lad);
```

Comment:
I am stating this is not an issue for master and 7.1.x, if you want to add an assert that is OK, but is it not required.